lib/rodauth/features/omniauth.rb

```diff
@@ -163,6 +163,14 @@ def omniauth_create_account?
       true
     end
 
+    def _before_omniauth_create_account
+      before_create_account
+    end
```

Comment:
Hi @jeremyevans, is this the correct way to provide default methods for `before` and `after` hooks?

Reply:
You would want to remove the existing method first, but that that is the correct method to override. I don't think it is a good idea to call hook method explicitly inside other hook methods by default, though.

The approach I would recommend, if the behavior in this pull request is desired, is for the omniauth code that creates accounts to call the `before_create_account` and `after_create_accounts` methods separately (before or after calling the `before_omniauth_create_account` and `after_omniauth_create_account` methods).  For backwards compatibility, that would likely have to be conditional and off by default, though.